nightly-style-policy.md

```diff
@@ -0,0 +1,26 @@
+Initial PR(s) implementing new syntax filed against rust-lang/rust should
+generally leave the rustfmt subtree untouched. In cases that that need to
+modify rustfmt (for example, to fix compiler errors that come from adding
+new variants to AST representation), they should modify rustfmt in such a
+way to keep existing formatting verbatim.
```

Comment:
Just for awareness, I will update this later with a direct link to a "how to" rustfmt doc on this once we have one, refs https://github.com/rust-lang/rustfmt/issues/5770